README.md

```diff
@@ -76,6 +76,54 @@ For VSCode, use these editor configuration flags:
         "--fast",
     ],
 ```
+## Development on kubernetes
+
+If you need to develop while gitops is running inside a kubernetes cluster you can use garden.io (Install Garden [Install garden-io](https://docs.garden.io/getting-started/1-installation))
+
+Garden version `0.12.26` was used to generate the configuration files for this repository.
+
+Only requirement is to run `gitops install` beforehand as need some kubernetes resources to be present in the cluster first.
+
+Once Garden is installed you just need to run:
+```
+garden deploy --dev wego-app-service
```

Comment:
Can we make this a `Makefile` target to ensure everyone users it the right way?

Reply:
@jpellizzari what would be a good name for the job? I'm thinking in `make dev` ?

Reply:
How about we change the current `make api-dev` target to `make local-dev` and add a new target that is `make cluster-dev`?